pagerduty/import_pagerduty_event_orchestration_path_global_test.go

```diff
@@ -0,0 +1,38 @@
+package pagerduty
```

Comment:
TODO: make sure tests aren't failing for accounts without proper entitlements

Reply:
Unfortunately it's hard to test warnings because in order to make features ineligible we need to use tokens for an account that doesn't have these features.